checker/manual/constant-value-checker.tex

```diff
@@ -49,11 +49,22 @@ \subsection{Type Annotations\label{constant-value-checker-type-annotations}}
 The set is limited to 10 entries; if a variable
 could be more than 10 different values, the Constant Value
 Checker gives up and its type becomes
-\refqualclass{common/value/qual}{UnknownVal} instead.
+\refqualclass{common/value/qual}{UnknownVal} instead (except for
+\refqualclass{common/value/qual}{IntVal}
+, see below)
 
 % \refqualclass{checker/value/qual}{BottomVal}, meaning that the expression
 % is dead or always has the value \<null>.
 
+In the case of
+\refqualclass{common/value/qual}{IntVal}
+with more than 10 possible values, instead of
+\refqualclass{common/value/qual}{UnknownVal}, its type becomes
+\refqualclass{common/value/qual}{IntRange}
+which is a type annotation with two integer arguments: \<from> and \<to>. It means that at run time, the expression only evaluates to a value that falls into the interval described by the two arguments. For example, an expression of type \<@IntRange(from = 0, to = 255)> evaluates to a value that falls between 0 and 255.
```

Comment:
Please break long lines in the .tex sources. This makes future edits and reviews easier, as git lists changes per line.